src/util/StreamerMode.cpp

```diff
@@ -25,46 +27,48 @@ const QStringList &broadcastingBinaries()
 bool isInStreamerMode()
 {
 #ifdef USEWINSDK
-    if (IsWindowsVistaOrGreater())
+    if (!IsWindowsVistaOrGreater() || !getSettings()->hideImagesInStreamerMode)
```

Comment:
This doesn't make sense here. Move it to the place in the code where it checks if it should display the images.

Reply:
After some changes in b9503b7 this fixed now.